app-api/src/test/java/org/openmrs/eip/app/route/sender/SenderActiveMqConsumerRouteTest.java

```diff
@@ -76,11 +77,13 @@ public void shouldProcessAndSaveASyncRequestMessage() {
 	@Test
 	public void shouldProcessAndSaveASyncResponseMessage() {
 		final String messageUuid = "message-uuid";
-		final LocalDateTime dateSent = LocalDateTime.now();
+		final LocalDateTime dateSent = DateUtilsTest.generateDateWithPrecision();
```

Comment:
Why can't you use LocalDateTime.now()?

Reply:
When i use LocalDateTime.now() without precision, some unit test fail.

Reply:
Does not make sense, because they were passing before

Reply:
Note that the database column has a precision of second but dateSent when you instantiate it has as a precision of nano seconds, so equals will fail if you compare them, please use LocalDateTime.of to create localdatetime instance with a lower precision

Reply:
OKay. I will remove that change.